### DISCUSSION TITLE
Disable »Remove Extension« on Firefox

### DISCUSSION THREAD

**Discussion by krautmitkohl** [proginosko/LeechBlockNG#463] [Q&A]
Hey everyone, I’ve been trying to get rid of the »Remove Extension« button in Firefox 127.0.1 for Mac.

There are two slightly different approaches I could find on the website and on the Tapatalk forum.
https://www.proginosko.com/leechblock/faq/hide-addon/
https://www.tapatalk.com/groups/leechblock/viewtopic.php?f=3&t=327&p=1096&hilit=%22remove+extension%22#p1096

I’ve followed the described procedure precisely.
However, the button is still there.

Any idea what I might have done wrong?
Have the methods stopped working?

**Comment by proginosko** [comment]
Did you go to **about:config** and set the _toolkit.legacyUserProfileCustomizations.stylesheets_ preference to **true**?

**Comment by proginosko** [comment]
I just realized that you're referring to a _button_ rather than a menu item. But I'm not sure which button that is. Where is it located? Can you post a screenshot?
